web/tests/buddy_data.test.cjs

```diff
@@ -410,7 +414,18 @@ test("show offline channel subscribers for small channels", ({override_rewire})
         me.user_id,
     ]);
 
-    override_rewire(narrow_state, "stream_id", () => stream_id);
+    const filter = new Filter([
+        {operator: "channel", operand: sub.stream_id},
+        {operator: "topic", operand: "Foo"},
+    ]);
+    message_lists.set_current({
+        data: {
+            filter,
+            participants: {
+                visible: () => new Set(),
+            },
+        },
+    });
```

Comment:
This is also a pattern that could potentially be replaced with a helper function in `web/tests/lib/` that does `new MessageListData` and `new MessageList` with the provided filter. It's OK to merge incremental improvements like this one, but I wanted to flag it since we're generally trying to not mock things without good reason.